tuned/plugins/plugin_net.py

```diff
@@ -9,6 +10,10 @@
 log = tuned.logs.get()
 
 WOL_VALUES = "pumbagsd"
+# re parsing result of "ip link show" command
+# just a first line, returns iname and qlen, we don't need more at the moment
+# if we would need other info, add named group with opt prefix
+RE_IP_LINK_SHOW = r"^\d+\: (?P<iname>[^\:\@]+)(?:\@[^\:]+)?\: <(?:.*)>(?:(?: +qlen +(?P<qlen>\d+))|(?: +\S+ +\S+))*$"
```

Comment:
Personally, I would use more relaxed RE not to be affected by possible future `ip` tool changes, e.g. something like (untested):
'\s*\d+\s*:\s+(?P<iname>[^:@]+)[^:]*.*\s+qlen\s+(?P<qlen>\d+)'

Do we need the iname? Then it could be even more relaxed:
'.*\s+qlen\s+(?P<qlen>\d+)'

This is not request for change, just comment.